### PR TITLE
Skip ClinVar variant submission for variants that are not loaded

### LINKED ISSUE #2180: Error while exporting to clinVar variants not loaded
Error:

```
Traceback (most recent call last):
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/utils.py", line 23, in decorated_function
    context = func(*args, **kwargs)
  File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/variant/views.py", line 263, in clinvar
    data = clinvar_export(store, institute_id, case_name, variant_id)
  File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/variant/controllers.py", line 389, in clinvar_export
    for gene in pinned_var.get("genes", []):
AttributeError: 'str' object has no attribute 'get'

```

The error happens for instance in this case:
https://scout.scilifelab.se/cust003/14032

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #2180 
The bug is caused by a function trying to submit to clinVar variants that are not loaded (variants that were once loaded for the case but disappeared after the case was re-runned).

**How to test on your local computer**:
-  On master branch.
-  Go to demo case and pin your favorite variant (POT1?).
-  Using mongodb compass, edit the case document and add some fake old variant to the list of the pinned variants. Like this:

image

The case pinned variants will look like this:
image

-  Add one phenotype term (HPO term) to the case so that it will be possible to create a clinVar submission.
-  Go back to the pinned variant (the existing one) and click the button `Submit to ClinVar`
-  Observe that the page crashed with the same error in #2180 
- Switch to this branch and repeat the last action (you might need to refresh the variant page)
- The ClinVar export shoud work, showing only the existing variant.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
- Tested adding the fake old variant:
image


- Submission using master crashes:
image


- Submission using this branch works:
image

**Comment by northwestwitch** [comment]
Ready for review

**Comment by mikaell** [comment]
> Ready for review

I'll have a look.

File: scout/server/blueprints/variant/controllers.py [hunk 1]
```diff
@@ -386,6 +386,9 @@ def clinvar_export(store, institute_id, case_name, variant_id):
 
     # gather missing transcript info from entrez (refseq id version)
     for pinned_var in pinned:
+        # Check if variant is not a previous variant that is not loaded in current case
```
Comment (mikaell) [inline scout/server/blueprints/variant/controllers.py:389]: Can you rephrase without the double negative? 

Otherwise code looks good!

Comment (northwestwitch) [reply scout/server/blueprints/variant/controllers.py]: Good idea!, fixed!

**Review by mikaell — APPROVED** [review]
